packages/elements-core/src/types.ts

```diff
@@ -116,6 +116,8 @@ export type Item = {
   uri: string;
 };
 
+export type RouterType = 'history' | 'memory' | 'hash' | 'static';
+
 export interface RoutingProps {
   /**
    * Only applies when using `history`-based routing. (See the `router` prop.) Specifies the base path under which
```

Comment:
Hey, this is a great PR!

Would you mind updating this doc description with `static` option? 🙏

Reply:
Sure.. happy to update the docs!